tfconfig/source_pos.go

```diff
@@ -26,14 +30,35 @@ func sourcePos(filename string, line int) SourcePos {
 	}
 }
 
+func sourceBlockHCL(block *hcl.Block) SourcePos {
+	pos := SourcePos{
+		Filename:  block.DefRange.Filename,
+		Line:      block.DefRange.Start.Line,
+		StartByte: block.DefRange.Start.Byte,
+		EndLine:   block.DefRange.End.Line,
+		EndByte:   block.DefRange.End.Byte,
+	}
+
+	body := block.Body.(*hclsyntax.Body)
+	if body != nil {
+		pos.EndByte = body.SrcRange.End.Byte
+		pos.EndLine = body.SrcRange.End.Line
+	}
```

Comment:
I assume the Body refers to the code within `{...}`
Would it make sense to create a new structure for the body range add this into a BodyPos sub-attribute?

`pos.BodyPos.Start = ...`

Reply:
Right that's body. I initially assumed that I would get first and last line number of the entire module call from the default range object, as when we look at the module call or resource objects it has attributes and everything from the body. But the range object just has the declaration start and end not the whole thing. So to match my expectation, I used declaration start & body end. I don't see "body" being separately called out in the paraded object before so that never occurred to me.